mitodl/release-script
# Add hash command to show latest commit info on servers

#### What are the relevant tickets?
Fixes #262 
Fixes #275 

#### What's this PR do?
Adds a command `hash` to show the latest commit hash and the corresponding log message. The command takes one argument: `ci`, `rc`, or `prod`. There's also a second commit to fix an error in Travis.

#### How should this be manually tested?
Run `python3 bot_local.py micromasters-eng hash ci`. In a short while you should see the commit message for the deployment.
